daisy_workflows/image_import/enterprise_linux/translate_rhel_8_licensed.wf.json

```diff
@@ -0,0 +1,78 @@
+{
+  "Name": "translate-rhel-8-licensed",
+  "Vars": {
+    "source_disk": {
+      "Required": true,
+      "Description": "The RHEL 8 GCE disk to translate."
+    },
+    "install_gce_packages": {
+      "Value": "true",
+      "Description": "Whether to install GCE packages."
+    },
+    "image_name": {
+      "Value": "rhel-8-${ID}",
+      "Description": "The name of the translated RHEL 8 image."
+    },
+    "family": {
+      "Value": "",
+      "Description": "Optional family to set for the translated image"
+    },
+    "description": {
+      "Value": "",
+      "Description": "Optional description to set for the translated image"
+    },
+    "import_network": {
+      "Value": "global/networks/default",
+      "Description": "Network to use for the import instance"
+    },
+    "import_subnet": {
+      "Value": "",
+      "Description": "SubNetwork to use for the import instance"
+    }
+  },
+  "Steps": {
+    "setup-disks": {
+      "CreateDisks": [
+        {
+          "Name": "disk-translator",
+          "SourceImage": "projects/compute-image-tools/global/images/family/debian-9-worker",
+          "SizeGb": "10",
+          "Type": "pd-ssd",
+          "FallbackToPdStandard": true
+        }
+      ]
+    },
+    "translate-disk": {
+      "Timeout": "2h",
```

Comment:
Remove the timeout so that workflow's default timeout is used. It's unlikely that translate will take more than 2hrs, but we thought the same for 60m timeout which proved to be incorrect. Also, other translate workflows have timeout removed from this step so this workflow would not be consistent with the rest.

Reply:
done